src/patterns.md

```diff
@@ -535,6 +533,8 @@ minimum to maximum value. The range of values for a `char` type are precisely th
 ranges containing all Unicode Scalar Values: `'\u{0000}'..='\u{D7FF}'` and
 `'\u{E000}'..='\u{10FFFF}'`.
 
+> **Edition Differences**: Before the 2021 edition, closed range patterns may be written using `...` in place of `..=`, with the same meaning.
```

Comment:
Can you clarify that before the 2021 edition `..=` is also allowed?

Reply:
I've reworded in b83fcc1